### PR TITLE
Implement responseURL property

### PR DESCRIPTION
**Pull request by dan-f**
This should implement #21 :)

### CODE REVIEW

File: test/src/events_test.coffee [hunk 1]
```diff
@@ -1,4 +1,4 @@
-describe.only 'XMLHttpRequest', ->
+describe 'XMLHttpRequest', ->
```
Comment (pwnall) [inline test/src/events_test.coffee]: This is a great catch, thank you very much!

File: test/src/redirect_test.coffee [hunk 2]
```diff
@@ -48,3 +48,10 @@ describe 'XMLHttpRequest', ->
         expect(headers).not.to.have.property 'content-length'
         done()
       @xhr.send 'This should be dropped during the redirect'
+
+    it 'provides the final responseURL', (done) ->
```
Comment (pwnall) [inline test/src/redirect_test.coffee]: This is the case that I was worried about. Thank you very much for the thorough tests!

**Comment by pwnall** [comment]
I released this in 0.1.4. Thank you very much for the great PR!

**Comment by dan-f** [comment]
No problem. Thank you for the quick merge and release!

**Pull request merged**